cli/src/commands/mod.rs

```diff
@@ -79,6 +79,8 @@ use crate::ui::Ui;
 enum Command {
     Abandon(abandon::AbandonArgs),
     Absorb(absorb::AbsorbArgs),
+    #[command(hide = true)]
+    Accuse(file::annotate::FileAnnotateArgs),
```

Comment:
You can add a comment pointing to <https://en.wikipedia.org/wiki/J%27Accuse...!>, to emphasize that we're being *historical* as opposed to merely mean.